src/scenes/UserProfile/queries.js

```diff
@@ -0,0 +1,54 @@
+import { gql } from 'apollo-boost'
+import { graphql } from 'react-apollo'
+
+import auth from '../../services/auth'
+
+export const getAllPersonalDataQuery = gql`
+  query getAllPersonalData($auth0UserId: String!) {
+    User(auth0UserId: $auth0UserId) {
+      createdAt
+      email
+      familyName
+      givenName
+      name
+      picture
+      flags {
+        id
+        createdAt
+        type
+        node {
+          question {
```

Comment:
You should add the attribute id of node and question in order to allow apollo to retrieve it from cache